src/games/strategy/triplea/delegate/Matches.java

```diff
@@ -121,6 +121,13 @@ public boolean match(final Unit unit) {
       return (ua.getIsCombatTransport() && ua.getIsSea());
     }
   };
+  public static final Match<Unit> UnitIsDefenselessTransport = new Match<Unit>() {
```

Comment:
I'll have a look at what you did for the raid one and see if I can do something similar.